src/Avalonia.Svg/AvaloniaModelExtensions.cs

```diff
@@ -117,61 +117,50 @@ public static AM.TextAlignment ToTextAlignment(this SKTextAlign textAlign)
         }
     }
 
-    public static AM.FontWeight ToFontWeight(this SKFontStyleWeight fontStyleWeight)
-    {
-        switch (fontStyleWeight)
-        {
-            default:
-            case SKFontStyleWeight.Invisible:
-                // TODO: FontStyleWeight.Invisible
-                throw new NotSupportedException();
-            case SKFontStyleWeight.Thin:
-                return AM.FontWeight.Thin;
-
-            case SKFontStyleWeight.ExtraLight:
-                return AM.FontWeight.ExtraLight;
-
-            case SKFontStyleWeight.Light:
-                return AM.FontWeight.Light;
-
-            case SKFontStyleWeight.Normal:
-                return AM.FontWeight.Normal;
-
-            case SKFontStyleWeight.Medium:
-                return AM.FontWeight.Medium;
-
-            case SKFontStyleWeight.SemiBold:
-                return AM.FontWeight.SemiBold;
-
-            case SKFontStyleWeight.Bold:
-                return AM.FontWeight.Bold;
-
-            case SKFontStyleWeight.ExtraBold:
-                return AM.FontWeight.ExtraBold;
-
-            case SKFontStyleWeight.Black:
-                return AM.FontWeight.Black;
-
-            case SKFontStyleWeight.ExtraBlack:
-                return AM.FontWeight.ExtraBlack;
-        }
-    }
-
-    public static AM.FontStyle ToFontStyle(this SKFontStyleSlant fontStyleSlant)
-    {
-        switch (fontStyleSlant)
-        {
-            default:
-            case SKFontStyleSlant.Upright:
-                // TODO: FontStyleSlant.Upright
-                return AM.FontStyle.Normal;
-            case SKFontStyleSlant.Italic:
-                return AM.FontStyle.Italic;
-
-            case SKFontStyleSlant.Oblique:
-                return AM.FontStyle.Oblique;
-        }
-    }
+    public static AM.FontWeight ToFontWeight(this SKFontStyleWeight fontStyleWeight) =>
+        // The numeric values in these two enumerations have the same scale.
```

Comment:
Please do not do unrelated refactoring in this PR, please keep changes simple to one feature otherwise reviewing is very hard and time consuming. Best to revert all unrelated changes in this PR.